### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by alejgarciarodriguez** [symfony-cli/symfony-cli#73]
I am getting this error 

`panic: runtime error: invalid memory address or nil pointer dereference
`
on ubuntu 20.04 new installation, after:

```
echo 'deb [trusted=yes] https://repo.symfony.com/apt/ /' | sudo tee /etc/apt/sources.list.d/symfony-cli.list
 sudo apt update
 sudo apt install symfony-cli
```

Version: 5.2.2

Then I try to create a project using:

`symfony new my-project`

I have permissions in the directory. This is only happening with 'new' command, others like console or composer are working. Well, I don't have php nor composer installed yet. Any ideas?

**Comment by fabpot** [comment]
Can you run the command with `-vvv` and paste the output here?

**Comment by alejgarciarodriguez** [comment]
Same output. `panic: runtime error: invalid memory address or nil pointer dereference`

**Comment by alejgarciarodriguez** [comment]
Same result with version 5.3.0

I tried in a docker container too (`docker run -it ubuntu bash`)

```bash
echo 'deb [trusted=yes] https://repo.symfony.com/apt/ /' | sudo tee /etc/apt/sources.list.d/symfony-cli.list
apt-get install ca-certificates # (required)
sudo apt update
sudo apt install symfony-cli
```

**Comment by camille-tabesto** [comment]
Same issue here. :(
Version 5.3.0 runing in WSL.

> Can you run the command with `-vvv` and paste the output here?

$ symfony new test -vvv
  panic: runtime error: invalid memory address or nil pointer dereference

**Comment by fabpot** [comment]
Looks like it happens when you don't have PHP installed. I will fix it, but in any case, you need PHP installed to make it work.

**Comment by camille-tabesto** [comment]
Can confirm it's working with PHP installed. Thank you !

**Comment by fabpot** [comment]
FYI, fixed by #79
